ntoskrnl/mm/section.c

```diff
@@ -1275,6 +1275,87 @@ MiReadPage(PMEMORY_AREA MemoryArea,
 }
 #endif
 
+static VOID
```

Comment:
Did you make any changes to this function? Perhaps moving it should be a separate commit to make that clear.

Reply:
No change. Ok for a separate commit.

Reply:
Done